core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4116,6 +4116,9 @@ definitions:
       status:
         $ref: '#/definitions/SubmissionStatus'
         description: Status
+      cost:
+        type: number
+        format: float
```

Comment:
nitpick: add a description for this one too

Reply:
Added on the line below 😄